### DISCUSSION TITLE
Failed to get a socket for multicast

### DISCUSSION THREAD

**Discussion by oloftangrot** [sstaub/Ethernet3#59] [Q&A]
I get a 0 as result when I try to call a udpmulticast. I started from the udp sender/listner example but the code fails already in the init method. Is there a working example sketch of how to use multicast?

**Comment by sstaub** [comment]
Sorry I know about the problem, but I didn't find a solution. Please use the original Ethernet library for using multicast.

**Reply by oloftangrot** [reply]
How do I find the original library?

**Comment by sstaub** [comment]
https://github.com/arduino-libraries/Ethernet

**Reply by oloftangrot** [reply]
I had got the impression that your fork was the only one that supported mulitcast. I seems like the Arduino reference is not up to date on this point. Anyway I tried to rebuild with the default Ethenet library and it works. I can see the IGMP registering using Wireshark and my recipient program gets data from an UNO module.

Thanks!
